lib/bgsite_official_web/templates/tag/show.html.leex

```diff
@@ -15,37 +13,32 @@
          end
          %>
        <div class="card-body text-center">
-
          <!-- <%= for tag <- website.tags do %>
-            <%= link tag.title, to: Routes.tag_path(@conn, :show, tag), class: "btn btn-dark btn-sm py-0 pl-0 pr-0 "%>
-
          <% end %> -->
-
          <%= if @current_admin do %>
-         <h5 class="card-title text-center"><%= link website.title , to: Routes.live_path(@conn, BgsiteOfficialWeb.WebsiteLive, website) %></h5>
+          <div class="float-right">
+          <a href="#" phx_click="like" phx-target=@myself>
+            <i class="far fa-heart"></i>
+            <p class="card-text "><small class="text-muted"><%= @likes %></small></p>
+          </a>
+         </div>
```

Comment:
IMO we dont need `<a href>` + `<div>`. Click event can be done on `<i class>` as well:
```
<i class="far fa-heart" phx-click="like"></i>
<p class="card-text "><small class="text-muted"><%= @likes %></small></p>
```